### PR TITLE
Remove the namespace parameter from cephClusterResource in details card

### PR DESCRIPTION
**Pull request by bipuladh**
/assign @cloudbehl 
This PR removes hard-coded namespace from Storage details since rook doesn't care about the namespace.

### CODE REVIEW

File: frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/details-card.tsx [hunk 1]
```diff
@@ -34,8 +33,6 @@ const infrastructureResource: FirehoseResource = {
 
 const cephClusterResource: FirehoseResource = {
   kind: referenceForModel(CephClusterModel),
-  namespaced: true,
```
Comment (cloudbehl) [inline frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/details-card.tsx]: @bipuladh we need to set the namespace to false if we want to skip the namespace.

Comment (bipuladh) [reply frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/details-card.tsx]: Noted.

**Comment by rawagner** [comment]
same resource is used in noobaa details card, should that be updated too ?

**Comment by cloudbehl** [comment]
@rawagner for Noobaa we are using for the version in detail card and the version is going to come from the OCS model which is not right now present. So will fix it when Neha's PR is In for the OCS install Operator which actually is bringing in the OCS model.

**Comment by afreen23** [comment]
> @rawagner for Noobaa we are using for the version in detail card and the version is going to come from the OCS model which is not right now present. So will fix it when Neha's PR is In for the OCS install Operator which actually is bringing in the OCS model.
> 
> But correction: The name of the noobaa should come from its CRD's. Right now using rook ceph CRD which is not correct.
> 
> @afreen23 ^^

The name for the Noobaa is actually the noobaa system name(coming from prometheus). The version is supposed to show OCS version for which we are using rook-ceph CRD now and as you mentioned that needs to be changed.
So, are you suggesting the name to come from crd and not from prometheus ?
@cloudbehl

**Comment by cloudbehl** [comment]
@afreen23 cool then we don't need to change anything in noobaa. The version needs to be changed for both and bipul is working on it.

**Pull request merged**